webapp/src/main/webapp/js/mc-view-components.js

```diff
@@ -299,31 +299,35 @@ MonitoringConsole.View.Components = (function() {
          let collapsed = false;
          for (let t = 0; t < model.groups.length; t++) {
             let group = model.groups[t];
-            let table = createTable(group);
-            collapsed = group.collapsed === true;
-            panel.append(table);
-            for (let r = 0; r < group.entries.length; r++) {
-               syntheticId++;
-               let entry = group.entries[r];
-               let type = entry.type;
-               let auto = type === undefined;
-               let input = entry.input;
-               if (entry.id === undefined)
-                 entry.id = 'setting_' + syntheticId;
-               entry.collapsed = collapsed;
-               if (type == 'header' || auto && input === undefined) {
-                  collapsed = entry.collapsed === true;
-                  table.append(createHeaderRow(entry));
-               } else if (!auto) {
-                  table.append(createRow(entry, createInput(entry)));
-               } else {
-                  if (Array.isArray(input)) {
-                    let [innerInput, innerSyntheticId] = createMultiInput(input, syntheticId, 'x-input');
-                    input = innerInput;
-                    syntheticId = innerSyntheticId;
-                  }
-                  table.append(createRow(entry, input));
-               }
+            if (group.available !== false) {
```

Comment:
Double negative?

Reply:
yes, this is so `true` and `undefined` evaluate to `true` but `false` is still `false`